python/lsst/ts/Dome/dome_csc.py

```diff
@@ -399,35 +423,27 @@ async def config_llcs(self, data):
         Parameters
         ----------
         data : `dict`
-            A dictionary with arguments to the function call. It should contain keys for all lower level
-            components to be configured with values that are dicts with keys for all the parameters that
-            need to be configured. The structure is::
+            A dictionary with arguments to the function call. It should contain a key "system" indicating
```

Comment:
I strongly recommend changing this to an API that allows specifying the "system" and "settings" as arguments:
```
async def config_llcs(self, system, settings):
```
That allows you to document the actual arguments, Python checks that they are spelled correctly, and the user does not have to needlessly make an extra struct to call this.

Reply:
There are two ways to do this: make settings a dict or make settings an array containing a single dict. I chose the first to keep things simple on the CSC side and up.